test/extensions/filters/http/common/compressor/compressor_filter_test.cc

```diff
@@ -174,7 +174,7 @@ TEST_F(CompressorFilterTest, DecodeHeadersWithRuntimeDisabled) {
   }
 }
 )EOF");
-  EXPECT_CALL(runtime_.snapshot_, getBoolean("foo_key", true)).WillOnce(Return(false));
+  EXPECT_CALL(runtime_.snapshot_, getBoolean("foo_key", true)).WillRepeatedly(Return(false));
```

Comment:
Nit: perhaps makes sense to have this expectation as
```
EXPECT_CALL(runtime_.snapshot_, getBoolean("foo_key", true)).Times(2).WillRepeatedly(Return(false));
```

to keep the number of `config_->enabled()` calls under control.